playbooks/roles/data/puppet_modules/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: 'Install puppet modules'
+  shell: puppet module install {{ item }}
+  with_items: '{{ data_puppet_modules }}'
+  tags:
+    - seed
+
+- name: 'Import puppet classes'
+  shell: >
+    hammer proxy import-classes --name '{{ data_puppet_modules_proxy_name }}'
```

Comment:
perhaps we could have a role that would set up hammer credentials for us? It would solve [this comment](https://github.com/theforeman/forklift/pull/398#discussion_r102865379) as well.